pkg/tap/protocol.go

```diff
@@ -59,3 +59,10 @@ type bessProtocol struct {
 func (s *bessProtocol) Stream() bool {
 	return false
 }
+
+type vfkitProtocol struct {
+}
+
+func (s *vfkitProtocol) Stream() bool {
```

Comment:
file should now be called protocols.go as it serves the use for multiple

Reply:
It was already handling multiple protocols before I added `vfkit` there, so that's a preexisting naming issue. I'll add a commit to rename it.

Reply:
I've now added the commit doing the rename.